pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -514,6 +518,17 @@ func findContainerStatusByName(status *v1.PodStatus, name string) (*v1.Container
 	return nil, fmt.Errorf("unable to find status for container with name %v in pod status (it may not be running)", name)
 }
 
+func StaticCPUPolicyConditionsSatisfied(cpuManagerPolicyName string, podQos v1.PodQOSClass, cpuRequest *apiresource.Quantity) bool {
```

Comment:
I'd use the same API proposed here: https://github.com/kubernetes/kubernetes/pull/117030/files#diff-c11864857fa45d5c2d29c3a85d753ccd2db31cca1c7fa046b7144e71f6e59029R407

I'd be fine with a different implementation, but the information I'd like to expose is if there are exclusive CPUs assigned to the container, rather than redo this logic.